doc/index.rst

```diff
@@ -1,8 +1,45 @@
-Plutus
-======
+Plutus Programming and Reference Guide
+=============================================
+
+About Plutus
+--------------------
+
+Plutus consists of Plutus Core, the programming, scripting and assembly language; 
+and Plutus Tx, the compiler that compiles the Haskell source code into Plutus Core 
+to form the on-chain part of a contract application. All of this is used in combination 
+to write Plutus scripts that run on the Cardano blockchain. Compiled Plutus Core 
+scripts or programs are able to interact with the Cardano ledger through the 
+ledger interface. 
+
+The Plutus Core Repository
+----------------------------------
+
+The `Plutus Core Repository <https://github.com/input-output-hk/plutus>`_ contains 
+the implementation, specification, and mechanized metatheory of Plutus Core. 
+It also contains Plutus Tx. 
+
+About This Documentation
+---------------------------------
+
+The purpose of this documentation is to serve as an introduction to the Plutus language 
+and to programming in Plutus. It is meant to be educational, so it includes explanations, 
+tutorials and how-to instructions. It is also intended as a reference for beginning and 
+experienced developers. 
+
+Intended Audience
+---------------------------
+
+The intended audience of this documentation includes DApp developers, Plutus developers, 
```

Comment:
These all seem to be the same group of people to me - people who need to implement smart contracts. Alternatively you can list what you assume of the audience, e.g., know how to program in some language; Haskell knowledge is helpful but not required, etc.

Reply:
What's a "Plutus developer"?

Reply:
@zliu41 Thanks for your comments. I've inserted "PlutusTx.List" as an example of a library.

Reply:
There is no such thing as a "Plutus script". There can be "Plutus Core scripts" or "Haskell programs which get compiled into Plutus Core".

Reply:
OK, thanks for that clarification.